src/main.rs

```diff
@@ -6,3 +8,5 @@ fn main() {
     let args = Args::parse();
     println!("{args:?}");
 }
+
+
```

Comment:
There are some extra newlines which have snuck in. This isn't a problem in our context just now, but may create some annoying, avoidable conflicts later